Dockerfile

```diff
@@ -1,17 +1,112 @@
-FROM jupyter/scipy-notebook:latest
+FROM jupyter/datascience-notebook:latest
 MAINTAINER ome-devel@lists.openmicroscopy.org.uk
 
 USER root
+
+## Swap the name of NB_USER
+RUN usermod -l omero $NB_USER && \
+    ln -s /home/$NB_USER /home/omero && \
+    mkdir -p /home/omero/data && \
+    chown omero /home/omero/data && \
+    chmod a+X /home/omero
+ENV NB_USER omero
+# Note: this replaces "OMERO_DATA_DIR=/home/omero/data bash -eux step02_all_setup.sh"
+
 RUN apt-get update -y && \
-    apt-get install -y nodejs
+    apt-get install -y nodejs wget git
+
+## Install R tools
+RUN pip install notedown
+RUN conda config --add channels bioconda && \
+    conda install --quiet --yes \
+    'r-mclust' \
+    'r-ggdendro' \
+    'r-igraph' \
+    'r-pheatmap'
+
+
+# FROM rOMERO-gateway/Dockerfile
```

Comment:
I'll check if there is one that's stable enough, but at the moment we're on a merge build.